Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -703,21 +706,30 @@ static void logProcessPoolState(const WebProcessPool& pool)
         "com.apple.WebKit.showPaintOrderTree"_s,
         "com.apple.WebKit.showRenderTree"_s,
         "com.apple.language.changed"_s,
+        "com.apple.mediaaccessibility.captionAppearanceSettingsChanged"_s,
+        "com.apple.powerlog.state_changed"_s,
         "com.apple.system.lowpowermode"_s,
+        "com.apple.system.timezone"_s,
+        "com.apple.zoomwindow"_s,
         "org.WebKit.lowMemory"_s,
         "org.WebKit.lowMemory.begin"_s,
         "org.WebKit.lowMemory.end"_s,
         "org.WebKit.memoryWarning"_s,
         "org.WebKit.memoryWarning.begin"_s,
         "org.WebKit.memoryWarning.end"_s,
     };
-    m_notifyTokens = WTF::compactMap(notificationMessages, [this](const ASCIILiteral& message) -> std::optional<int> {
+    m_notifyTokens = WTF::compactMap(notificationMessages, [weakThis = WeakPtr { *this }](const ASCIILiteral& message) -> std::optional<int> {
         int notifyToken = 0;
-        auto status = notify_register_dispatch(message, &notifyToken, dispatch_get_main_queue(), ^(int token) {
-            if (!m_processes.isEmpty()) {
+        auto status = notify_register_dispatch(message, &notifyToken, dispatch_get_main_queue(), [weakThis, message](int token) {
+            RefPtr strongThis = weakThis.get();
```

Comment:
We don't use the `strong` naming with C++ classes. We only use `strong` in OjbC. Please rename to `protectedThis`.